tests/x11/yast2_lan_restart.pm

```diff
@@ -62,20 +62,17 @@ sub change_hw_device_name {
     assert_screen 'yast2_lan_hardware_tab';
     send_key 'alt-e';        # Change device name
     assert_screen 'yast2_lan_device_name';
-    send_key 'alt-m';        # Udev rule based on MAC
-    send_key 'tab';
-    send_key 'tab';
-    send_key 'tab';
-    send_key 'tab';
-    type_string "$dev_name";
+    send_key 'tab' for (1 .. 2);
+    type_string $dev_name;
+    wait_screen_change { send_key 'alt-m' };    # Udev rule based on MAC
     save_screenshot;
-    send_key 'alt-o';
-    send_key 'alt-n';
+    send_key $cmd{ok};
+    send_key $cmd{next};
```

Comment:
We have too many key presses without single screen check, I guess it will keep instability of the test. Don't you mind to improve this code? Or at least verify that it's stable (arm workers would be a good for stability tests as are not performing well).

Reply:
sure, let me check for arm, I did it on purpose as I think that this order avoid more needles. Let's see :)

Reply:
I guess it will be enough to have one that failure is obvious, if it's stable, we can postpone having a hope to get YaST UI testing framework in foreseeable future =)

Reply:
Added VR for arm in the 1st comment.